RickAndMorty-iOS/Views/Locations/LocationDetailsView.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  LocationDetailsView.swift
+//  RickAndMorty-iOS
+//
+//  Created by Calvin Pak on 2023-03-14.
+//
+
+import UIKit
+import SnapKit
+
+class LocationDetailsView: UIView {
+
+    lazy var collectionView: UICollectionView = {
+        let collectionView = UICollectionView(frame: self.bounds, collectionViewLayout: createLayout())
+        collectionView.register(HeaderView.self,
+                                forSupplementaryViewOfKind: UICollectionView.elementKindSectionHeader,
+                                withReuseIdentifier: "HeaderView")
+        collectionView.register(InfoCell.self,
+                                forCellWithReuseIdentifier: InfoCell.identifier)
+        collectionView.register(CharacterRowCell.self,
+                                forCellWithReuseIdentifier: CharacterRowCell.identifier)
+
+        return collectionView
+    }()
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+    }
+
+    init() {
+        super.init(frame: .zero)
+        setupViews()
+        setupConstraints()
+    }
+
+    private func setupViews() {
+        self.backgroundColor = .systemBackground
+        self.addSubview(collectionView)
+        collectionView.accessibilityIdentifier = "LocationDetailsCollectionView"
```

Comment:
accessibilityIdentifier move to a separate enum and use them in tests target and app target.